### DISCUSSION TITLE
How do you automatically download a configuration with the permalink?

### DISCUSSION THREAD

**Discussion by ghost** [Dav1dde/glad#455] [Q&A] [answered]
I already have the permalink but my question is how would you automatically generate it? All the permalink does is get the configuration, which is helpful but as I said I want to automatically generate it also

Any help would be appreciated

**Comment by Dav1dde** [comment, marked as answer]
There are no permalinks, every link will expire within 24 hours. Also please don't use the website for that, it may be down or you might hit a Cloudflare rule and will get IP banned.

You should download once and check the generated files in with the rest of your code.

If you want to automatically generate the code, you can use `glad` as a commandline tool, it can be installed via pip, e.g. `pip install glad2`.

**Reply by ghost** [reply]
Yeah thats kind of what I did, I just made a copy of your project with the options i wanted and downloaded it and put it as a repo for reuse
